styles/admin.css

```
@@ -231,14 +231,19 @@
 
   /* Sidebar */
   .admin-sidebar {
-    @apply sticky top-0 z-40 flex h-screen w-[82px] shrink-0 flex-col justify-between overflow-y-auto border-r px-2 pb-5 pt-8 sm:w-[248px] sm:px-5 sm:pt-10 lg:w-[264px];
+    @apply sticky top-0 z-40 hidden w-[248px] shrink-0 flex-col border-r px-5 pt-8 md:flex lg:w-[264px] lg:pt-10;
+    height: 100vh;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)